src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -22,7 +22,7 @@ const AuthenticatorRow = View.extend({
           </div>\
         </div>\
         <div class="authenticator-description">\
-          <h3 class="authenticator-label">{{label}}</h3>\
+          <h3 class="authenticator-label no-translate">{{label}}</h3>\
```

Comment:
The intention here is to use the `displayName` of the authenticator the user is enrolling or challenging, or falling back to v1 customizations (see Duo and Symantec authenticators). Given that this is a DB stored value configured by the user/admin - we won't be able to guarantee this will be translated.